github/orgs.go

```diff
@@ -213,3 +219,25 @@ func (s *OrganizationsService) Edit(ctx context.Context, name string, org *Organ
 
 	return o, resp, nil
 }
+
+// List installations for an organization.
+//
+// GitHub API docs: https://developer.github.com/v3/orgs/#list-installations-for-an-organization
+func (s *OrganizationsService) ListInstallations(ctx context.Context, org string) (*OrganizationInstallations, *Response, error) {
```

Comment:
I think you'll need to pass in and process a `*ListOptions` to handle pagination.